<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_addEps</class>
 <widget class="QDialog" name="Dialog_addEps">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>354</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>354</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>354</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Adding Episode</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog_addEps {
background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #f857a6, stop:1 #ff5858);
}</string>
  </property>
  <widget class="QLabel" name="label_activity">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_activity {
color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>Adding Episode</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>251</x>
     <y>200</y>
     <width>123</width>
     <height>64</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QRadioButton" name="radioButton_card1">
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
width: 12px;
height: 12px;
border-radius: 6px;
}
QRadioButton::indicator:checked {
background-color: #ffd700;
border: 1px solid blue;
}
QRadioButton::indicator:unchecked {
background-color: white;
border: 1px solid blue;
}
QRadioButton {
color: #ffffff;
font-weight: bold;
font-size: 16px;
}
</string>
      </property>
      <property name="text">
       <string>Golden Card</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="radioButton_card2">
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
width: 12px;
height: 12px;
border-radius: 6px;
}
QRadioButton::indicator:checked {
background-color: #ffd700;
border: 1px solid blue;
}
QRadioButton::indicator:unchecked {
background-color: white;
border: 1px solid blue;
}

QRadioButton {
color: #ffffff;
font-weight: bold;
font-size: 16px;
}</string>
      </property>
      <property name="text">
       <string>Golden Card</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>161</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_upCont">
      <property name="styleSheet">
       <string notr="true">#label_upCont {
color: #ffffff;
font-weight: bold;
font-size: 14px;
}</string>
      </property>
      <property name="text">
       <string>Continent Name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_contName"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>161</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_upCtr">
      <property name="styleSheet">
       <string notr="true">#label_upCtr {
color: #ffffff;
font-weight: bold;
font-size: 14px;
}</string>
      </property>
      <property name="text">
       <string>Country name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_ctrName"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>341</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_date">
      <property name="styleSheet">
       <string notr="true">#label_date {
color: #ffffff;
font-weight: bold;
font-size: 13px;
}</string>
      </property>
      <property name="text">
       <string>First aired (&quot;dd/MM/yyyy&quot;)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_date"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_city">
      <property name="styleSheet">
       <string notr="true">#label_city {
color: #ffffff;
font-weight: bold;
font-size: 14px;
}</string>
      </property>
      <property name="text">
       <string>City</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_city"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_name1">
        <property name="styleSheet">
         <string notr="true">#label_name1 {
color: #ffffff;
font-weight: bold;
font-size: 16px;
}</string>
        </property>
        <property name="text">
         <string>Host 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_fio1"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_name2">
        <property name="styleSheet">
         <string notr="true">#label_name2 {
color: #ffffff;
font-weight: bold;
font-size: 16px;
}</string>
        </property>
        <property name="text">
         <string>Host 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_fio2"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_sum">
      <property name="styleSheet">
       <string notr="true">#label_sum {
color: #ffffff;
font-weight: bold;
font-size: 14px;
}</string>
      </property>
      <property name="text">
       <string>Total Sum</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_summa"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>265</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="styleSheet">
       <string notr="true">#pushButton_cancel {
width: 100px;
height: 35px;
}</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_OK">
      <property name="styleSheet">
       <string notr="true">#pushButton_OK {
width: 100px;
height: 37px;
border-style: solid;
border-width: 2px;
border-color: #ffffff;
color: #ffffff;
background-color: #2E85D3;
font-weight: bold;
}
#pushButton_OK:hover {
background-color: #7fbdf5;
border-color: #b8ffbd;
}
</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_podCard1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>208</y>
     <width>17</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
border-radius: 8px;
background-color: blue;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_podCard2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>238</y>
     <width>17</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
border-radius: 8px;
background-color: blue;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_podCard2</zorder>
  <zorder>label_podCard1</zorder>
  <zorder>label_activity</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
